Resources/doc/handlers.md

```diff
@@ -48,7 +48,27 @@ The message can access the filled in data of the form, for example submitted fie
 {
     "from": "{{ data.email }}", 
     "subject": "Email Form subject", 
-    "body": "{{ body|json_encode }}"
+    "body": "{{ body|json_encode }}",
+}
+```
+
+Its the same for the attachments with 1 attachment per line.
```

Comment:
I would rather state: "To include one or multiple attachments to your email, declare them to a variable as shown below."